backend/src/main/java/stand/lemonade/service/WalletServiceImpl.java

```
@@ -15,12 +16,11 @@
 	@Inject
 	private WalletDal walletDal;
 	
-	public WalletServiceImpl() {
-	}
+	public WalletServiceImpl() { }
 
 	@Transactional
 	@Override
-	public List<WalletModel> getAllWallets() {
+	public List<WalletModel> getAllWallets() throws LemonadeException {
```

why would this method need to throw an exception?